example/lib/example_icon.dart

```diff
@@ -15,6 +17,7 @@ class ExampleIcon implements Comparable {
       other is ExampleIcon &&
           runtimeType == other.runtimeType &&
           iconData == other.iconData &&
+          duotoneIconData == other.duotoneIconData &&
```

Comment:
I don't like this approach. Null-safety is a feature in dart, but this introduces exactly those pitfalls it is supposed to prevent: what happens if both are null? For me, it's also not clear what exactly happens here.

Reply:
The intention here is to update the implementation of `Comparable` - since there is a new property (`duotoneIconData`), it has to be compared against the property of the same name in `other` in order to establish if the two items are truly equal or not.

> what happens if both are null?

I'm not sure I follow on why this is is an issue - if both the current `ExampleIcon` and `other` (which is cast as an `ExampleIcon`) have `null` `iconData` or `duotoneIconData`, then that particular comparison just returns as `true`.  If you mean what happens when both `iconData` and `duotonIconData` are `null`, that shouldn't affect the comparison, either - either the `other` `ExampleIcon` also has both fields as `null`, or it doesn't.